frontends/concrete-python/concrete/fhe/compilation/server.py

```diff
@@ -166,33 +163,23 @@ def create(
                 set_llvm_debug_flag(True)
             if configuration.compiler_verbose_mode:  # pragma: no cover
                 set_compiler_logging(True)
-            if configuration.jit:  # pragma: no cover
```

Comment:
Could you remove `jit` option from the configuration as well? `conftest.py` needs to be updated to stop using JIT as well.

Reply:
Ah yes done !